src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
-export interface Figure {
+export type Shape = 'triangle'|'circle'|'rectangle';
+export type Color = 'red'|'green'|'blue';
 
+export interface Figure {
+  shape: string;
+  color:string;
+  getArea():number;
```

Comment:
You forgot spaces between : and type
Check it in whole document
Check it with linter maybe